default.yml

```diff
@@ -27,21 +27,24 @@ Style/NumericLiterals:
 Style/HashSyntax:
   EnforcedShorthandSyntax: never
 
+Layout/LineLength:
+  Max: 140
```

Comment:
Why? I am always against increasing the length, as it's always a moving goalpost. In a few months, you'd want 150, then 160 and so it goes

Reply:
@doutatsu agree, let's keep it 120 👍